.gitignore

```diff
@@ -55,3 +55,31 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Jupyter and iPython notebook checkpoints
+*.ipynb_checkpoints
+
+# Spyder app workspace config file
+.spyderworkspace
+
+# PyCharm environment files
+.idea/
+
+# VSCode environment files
+.vscode/
+*.code-workspace
+
+# Apple OSX
+*.DS_Store
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# pythonenv
+pythonenv*/
```

Comment:
This seems to be not a generic pattern to be ignored.

Reply:
My understanding is this is the files for the python environments, are we sure don't want to ignore this

Reply:
Python virtualenv folder names are arbitrary. You can't expect anyone to name their folder "pythonenv".

Reply:
I will delete it but we should also delete it form the main repo

Reply:
Yes indeed. It was added in https://github.com/projectmesa/mesa/commit/0f1bcdc0f441b13a0b4dfd9858d7c63bc67f56bc.

Reply:
fixed